examples/CMakeLists.txt

```diff
@@ -17,8 +17,9 @@ set(ENV{PKG_CONFIG_PATH} "${PKG_CONFIG_CHANGED_PATH}")
 
 find_package(LibtorrentRasterbar REQUIRED)
 find_package(Boost REQUIRED COMPONENTS system)
+find_package(OpenSSL REQUIRED)
```

Comment:
I'm not completely familiar with the cmake file, but I would imagine this to be an options build configuration, primarily because it can be quite a lot of work to build or "install" openssl on windows. There's no option to enable openssl right now?